bot/exts/moderation/infraction/infractions.py

```diff
@@ -10,7 +10,7 @@
 from bot import constants
 from bot.bot import Bot
 from bot.constants import Event
-from bot.converters import Duration, Expiry, MemberOrUser
+from bot.converters import Duration, Expiry, MemberOrUser, UnambiguousMemberOrUser
```

Comment:
Why is `MemberOrUser` still used for bans?
https://github.com/python-discord/bot/blob/2018cb6e5386ead7609b6dcb065abaf30d32fb5a/bot/exts/moderation/infraction/infractions.py#L329-L342
https://github.com/python-discord/bot/blob/2018cb6e5386ead7609b6dcb065abaf30d32fb5a/bot/exts/moderation/infraction/infractions.py#L388-L390

Reply:
That's not the ban command, it's a helper function, here it's used for type-hinting and they'd both do the exact same thing.